src/internal/modules/account/helpers.js

```diff
@@ -0,0 +1,33 @@
+const { throwIfError } = require('@envage/hapi-pg-rest-api');
+const internalConfig = require('internal/config');
+const services = require('internal/lib/connectors/services');
+
+const getEmailRegex = (config = internalConfig) => {
```

Comment:
If this ends up in a file all by itself it could move back to the form, as I think it is only used there